### PR TITLE
Task4597 - Refactor vuetify Overlay docs

### PR DESCRIPTION
**Pull request by MrShmiTT**
The overlay chapter is refactored.

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -1591,42 +1591,84 @@ For examples of usage see: [JDI Vuetify Lists tests](https://github.com/jdi-test
...
+  </v-app>
+</div>
+```
 
-For examples of usage see: [JDI Vuetify Overlays tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/OverlayTests.java)
+| Method                    | Description                              | Return Type   |
+|---------------------------|------------------------------------------|---------------|
+| **absolutePosition()**    | Returns object for work with assertions  | OverlayAssert |
+| **backgroundColor()**     | Gets background color                    | String        |
+| **has()**                 | Returns object for work with assertions  | TooltipAssert |
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:1661]: Why Tooltip? :)

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -1591,42 +1591,84 @@ For examples of usage see: [JDI Vuetify Lists tests](https://github.com/jdi-test
...
+| **hover()**               | Hovers mouse cursor over the element     | void          |
+| **isAbsolute()**          | Shows that required element is absolute  | boolean       |
+| **isDisplayed()**         | Shows that required element is displayed | boolean       |
+| **is()**                  | Returns object for work with assertions  | TooltipAssert |
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:1665]: Why Tooltip? :)

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (MrShmiTT) [reply source/includes/documentation/_vue_js_elements.md:1661]: Missprint :)
Fixed

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (MrShmiTT) [reply source/includes/documentation/_vue_js_elements.md:1665]: Missprint :)
Fixed

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**